docs/bidirectional-communication.md

```diff
@@ -0,0 +1,80 @@
+# Bi-directional Communication
+
+The SDK implements bi-directional communication between the Vera platform and the client
+application.
+
```

Comment:
On iOS we mention this:
https://github.com/resonai/vera-ios-sdk/blob/readme/docs/bidirectional-communication.md#events

Should this also be covered here?

Reply:
As I know in Android we don't have them yet(